pkg/oomstore/types/config.go

```diff
@@ -27,14 +27,13 @@ type OomStoreConfig struct {
 	OfflineStore  OfflineStoreConfig  `yaml:"offline-store"`
 	OnlineStore   OnlineStoreConfig   `yaml:"online-store"`
 
-	StreamPushProcessor *StreamPushProcessorConfig `yaml:"stream-push-processor"`
+	PushProcessor *PushProcessorConfig `yaml:"push-processor"`
 }
 
-type StreamPushProcessorConfig struct {
-	RevisionInterval time.Duration `yaml:"revision-interval"`
-	Period           time.Duration `yaml:"period"`
-	MinPeriod        time.Duration `yaml:"min-period"`
+type PushProcessorConfig struct {
 	BufferSize       int           `yaml:"buffer-size"`
+	FlushInterval    time.Duration `yaml:"flush-interval"`
+	RevisionInterval time.Duration `yaml:"revision-interval"`
```

Comment:
what's the difference between `FlushInterval` and `RevisionInterval`?

Reply:
btw, I remember we agree on `SnapshotInterval`?

Reply:
`RevisionInterval` will be remove once @lianxmfor added `SnapshotInterval` in group metadata.
`FlushInterval` literally means interval between flushes.